js/app/modules/arrangement/sideBySide.js

```diff
@@ -12,7 +12,7 @@ const Knowledge = imports.app.knowledge;
 const Module = imports.app.interfaces.module;
 const Utils = imports.app.utils;
 
-const SPACING_MAX = 50;
+const SPACING_MAX = 0;
```

Comment:
This disables the responsive spacing between categories, did the design changed since I implemented the changes in the topbar?